packages/google_adsense/lib/src/ad_unit_widget_web.dart

```diff
@@ -77,7 +75,10 @@ class AdUnitWidgetWeb extends AdUnitWidget {
 class _AdUnitWidgetWebState extends State<AdUnitWidgetWeb>
     with AutomaticKeepAliveClientMixin {
   static int adUnitCounter = 0;
-  double adHeight = 1.0;
+  // Make the ad as wide as the available space, so Adsense delivers the best
+  // possible size.
+  Size adSize = const Size(double.infinity, 1);
```

Comment:
Would like to know the difference here between not specifying width at all. Is it just for readability?

Reply:
I think this is also needed so the ad is not overlaid on top of flutter content, and instead it *pushes* flutter things around it. (Did you mean the height?)

Reply:
No, I mean width. 

Height is important becase it pushes other wisgets around and allow ad to be visible (If I recall correclty, ad was not overlaying on top of other content, but otherwise was cut off, kind of displayed "under" surrounding Flutter widgets).

But for width my expectation would be that it will simply take all the available space anyways, if I understand [SizedBox](https://api.flutter.dev/flutter/widgets/SizedBox-class.html) behavior correctly.

As agreed merging it as is, but would love to discuss